src/common/github-flavored-markdown.js

```diff
@@ -1,19 +1,42 @@
 import memoizee from 'memoizee';
 import MarkdownIt from 'markdown-it';
+import replaceLink from 'markdown-it-replace-link';
 
-const MAX_MEMOIZED_VALUES = 1000;
-
-const gfm = new MarkdownIt();
-const render = memoizee(gfm.render.bind(gfm), { max: MAX_MEMOIZED_VALUES });
-const renderInline = memoizee(gfm.renderInline.bind(gfm), { max: MAX_MEMOIZED_VALUES });
+const getGfmForCdnRootUrl = memoizee(cdnRootUrl => {
+  let gfm;
+  if (cdnRootUrl) {
+    gfm = new MarkdownIt({
+      replaceLink: link => link.startsWith('cdn://') ? `${cdnRootUrl}/${link.substr(6)}` : link
+    });
+    gfm.use(replaceLink);
+  } else {
+    gfm = new MarkdownIt();
+  }
 
+  return gfm;
+});
 class GithubFlavoredMarkdown {
-  render(markdown) {
-    return render(markdown);
+  render(markdown, { cdnRootUrl } = {}) {
+    return getGfmForCdnRootUrl(cdnRootUrl).render(markdown);
```

Comment:
I guess what I was missing is that this replacement is done on the fly on render, right?

Reply:
Yes this is the reverse process on render